### PR TITLE
add /etc -> /host/etc volume bind

### PR DESCRIPTION
**Pull request by deepskyblue86**
## What this PR does / why we need it:
After integrating falcosecurity/libs/pull/220, the agent can have an up-to-date users and groups list from `/etc/passwd` and `/etc/group`.
To have those files in the container we mount `/etc` in `/host/etc` and symlink them in the container `/etc`.
If we mount them individually, they will become stale in case of an inode change, which is what happens with `useradd`.

## Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [ ] Title of the PR starts with chart name (e.g. [mychartname])
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md (or README.tpl in some charts)
- [ ]  Check GithubAction checks (like lint) to avoid merge-check stoppers

Check Contribution guidelines in README.md for more insight.

### CODE REVIEW

File: charts/agent/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,9 @@
 
 This file documents all notable changes to the Sysdig Agent Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
 
+### Minor changes
```
Comment (achandras) [inline charts/agent/CHANGELOG.md:9]: Please add version heading above

File: charts/agent/templates/daemonset.yaml [hunk 2]
```diff
@@ -175,6 +175,9 @@ spec:
               name: modprobe-d
               readOnly: true
             {{- end }}
+            - mountPath: /host/etc
```
Comment (achandras) [inline charts/agent/templates/daemonset.yaml:175]: with this addition I think the `osrel` and `etc-fs` mounts/volumes can be removed

File: charts/sysdig/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,9 @@
 
 This file documents all notable changes to Sysdig Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
 
+### Minor changes
```
Comment (achandras) [inline charts/sysdig/CHANGELOG.md:8]: version heading

**Comment by achandras** [comment]
Each chart's `Chart.yaml` will need a minor version bump

**Review by lilx1ao — APPROVED** [review]

**Pull request merged**